util/options.cc

```diff
@@ -691,6 +711,9 @@ font_options_t::get_font (void) const
   if (font_size_y == FONT_SIZE_UPEM)
     font_size_y = hb_face_get_upem (face);
 
+  hb_font_set_ppem (font, x_ppem, y_ppem);
+  hb_font_set_ptem (font, ptem);
```

Comment:
Should I guard this against default value? As comparing double to zero is not that nice, what would be the best way? As tests are not failing that perhaps not that needed?

Reply:
The defaults match the font's defaults, so it's harmless to set. Also, no problem comparing double to zero when you specifically set it to zero yourself...